### DISCUSSION TITLE
how to display date format in grid view?

### DISCUSSION THREAD

**Discussion by XR97** [pawelsalawa/sqlitestudio#4246] [General]
### Details
I imported a datatable to sqlite, and viewed the data by sqlitestudio. And I found the date class column in sqlite grid view displayed integer. As the following picture... So how to display the date format in grid view?

![image](https://user-images.githubusercontent.com/46560490/139016373-393cdfe9-2fa1-40c7-bf9a-6f389286d994.png)

![image](https://user-images.githubusercontent.com/46560490/139016405-cdd9b268-98cb-4b30-a0f9-cc06e17017df.png)

### Operating system
windows10

### SQLiteStudio version 
sqlitestudio-3.3.3.zip

**Comment by pawelsalawa** [comment]
SQLite databases are not strict with data types. You can declare `date` type and still put whatever data you want (integers, strings) into that column (In upcoming SQLite 3.37 there will be `STRICT` tables, who enforce data types - https://sqlite.org/draft/stricttables.html).

It seems that your database stores dates as integers and then these values are somehow converted to date values by the application that runs on this database.

SQLiteStudio only shows what is there in the database. If it shows integers, it means you have integers in the database.
